core/src/test/java/org/infinispan/eviction/impl/EvictionWithPassivationTest.java

```diff
@@ -46,8 +50,8 @@ public EvictionWithPassivationTest() {
    @Factory
    public Object[] factory() {
       return new Object[] {
-            new EvictionWithPassivationTest().withStorage(StorageType.BINARY),
-            new EvictionWithPassivationTest().withStorage(StorageType.OBJECT),
+//            new EvictionWithPassivationTest().withStorage(StorageType.BINARY),
```

Comment:
Are these going to stay uncommented ?

Reply:
No, I have to go back through and fix some tests back. I just happen to miss undoing this one before - commented out while testing to more easily reproduce.